package.json

```diff
@@ -11,6 +11,7 @@
     "courses-gen": "npm run ts-node -- src/requirements/courses-json-generator.ts",
     "req-gen": "npm run ts-node -- src/requirements/requirement-json-generator.ts",
     "track-users": "npm run ts-node -- scripts/track-users.ts",
+    "courses-populate": "npm run ts-node -- scripts/migration/courses-populate.ts",
```

Comment:
There is a one-liner PR in that updates the path for this script from `migration/courses-populate.ts` to `population/courses-populate.ts`

Reply:
Make sure that gets in before we merge this thing